astronomer/providers/snowflake/sensors/snowflake.py

```diff
@@ -63,6 +63,7 @@ def __init__(
         super().__init__(
             conn_id=snowflake_conn_id,
             sql=sql,
+            parameters=parameters,
```

Comment:
Would they have not been passed already as part of **kwargs that get passed to the parent class?

Reply:
Unfortunately not. `parameters` are extracted as their own keyword arg (as they are in SqlSensor too) and wouldn't be part of arbitrary `**kwargs` when being passed to the parent.

Hopefully to illustrate, `ChildOne` inherits from `ParentOne` and `parameters` are not passed to `ParentOne`. Both `kwargs` and `parameters` are empty dicts.
```
>>>
>>> class ParentOne:
...     def __init__(self, parameters=None, **kwargs) -> None:
...         self.parameters = parameters or {}
...         self.kwargs = kwargs
...
>>>
>>> class ChildOne(ParentOne):
...     def __init__(self, parameters, **kwargs) -> None:
...         super().__init__(**kwargs)
...
>>>
>>> c1 = ChildOne(parameters={"param1": 123})
>>> print(c1.parameters)
{}
>>> print(c1.kwargs)
{}
```
Now a similar setup but `ChildTwo` passes `parameters` to `ParentTwo`. `ParentTwo.parameters` is what was initialized when originally calling `ChildTwo`.
```
>>>
>>>
>>> class ParentTwo:
...     def __init__(self, parameters=None, **kwargs) -> None:
...         self.parameters = parameters or {}
...         self.kwargs = kwargs
...
>>>
>>> class ChildTwo(ParentTwo):
...     def __init__(self, parameters, **kwargs) -> None:
...         super().__init__(parameters=parameters, **kwargs)
...
>>>
>>> c = ChildTwo(parameters={"param1": 123})
>>> print(c.parameters)
{'param1': 123}
>>> print(c.kwargs)
{}
>>>
```

Reply:
oops sorry I totally misread the the existing implementation. I thought we were not having an existing parameters param and were newly adding there. 

I thought we had a setup like this
```
class A(base):
    def __init__(self, parameters=None, **kwargs):
        self.parameters = parameters
        print("A parameters", parameters)
        super().__init__(**kwargs)

class B(A):
    def __init__(self, **kwargs):
        print("B")
        super().__init__(**kwargs)


b = B(parameters="test")
```

din't realise B already defined parameters and extracted from kwargs.